specifyweb/frontend/js_src/lib/components/QueryBuilder/helpers.ts

```diff
@@ -322,21 +324,15 @@ export function smoothScroll(element: HTMLElement, top: number): void {
   else element.scrollTop = element.scrollHeight;
 }
 
-export function isSpecial(
+const containsOr = (
```

Comment:
A good habit to get into:
Put the parent function first in the file, followed by the child functions that are called from inside the parent.
This way, natural code reading direction matches the direction in which code is executed (i.e, put isQueryModern first, followed by containsOr and the rest)